bin/pycbc_multi_inspiral

```diff
@@ -349,10 +349,8 @@ with ctx:
             # Only do this is idx doesn't get time shifted out of the time
             # we are looking at.
             idx_dict = {
-                ifo: idx[ifo][np.logical_and(
-                    idx[ifo] > time_delay_idx[ifo],
-                    idx[ifo] - time_delay_idx[ifo] < len(snr_dict[ifo])
-                    )
+                ifo: idx[ifo][
+                    idx[ifo] + time_delay_idx[ifo] < len(snr_dict[ifo]) - 1
```

Comment:
Since I am not familiar with the code around this change, what is the reason for the change? Was the old logic incorrect? Should the code also check that `idx[ifo] + time_delay_idx[ifo] >= 0`?

Reply:
The time delay was previously applied using a numpy operation. It was changed so that time delay is now added as necessary. The logic check here was overlooked when that change was made, so when I was running workflow with many triggers it would try and run chi-squared tests on triggers with indices outside of the data. There certainly could be cases where more logic checks are necessary.